packages/solo/src/vat-http.js

```diff
@@ -3,8 +3,15 @@ import { E } from '@endo/eventual-send';
 import { Far } from '@endo/marshal';
 import { assert, details as X } from '@agoric/assert';
 import { getReplHandler } from '@agoric/vats/src/repl.js';
+import { makePromiseKit } from '@endo/promise-kit';
 import { getCapTPHandler } from './captp.js';
 
+const introduceWith = f => {
```

Comment:
Nit: This function seems a little mysterious on its own, maybe name it something more specific to attMaker or just do it inline.